tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Net.cs

```diff
@@ -121,6 +121,12 @@ public class Net : LanguageBase
             };
         }
 
+        private string GetProfileOutputFile(string testName, string arguments)
+        {
+            var fileName = $"{testName}_{arguments.Replace(" --", "_").Replace("--", "_").Replace(" ", "-")}_{DateTime.Now.ToString("yyyyMMdd_hhmmss")}.nettrace";
```

Comment:
You may also want to add the version of the primaryPackage to determine if the profile is for `source` or a GA version like `12.14.1`.